test/integration/main.go

```diff
@@ -73,6 +74,11 @@ func KicDriver() bool {
 	return strings.Contains(*startArgs, "--driver=docker") || strings.Contains(*startArgs, "--vm-driver=docker") || strings.Contains(*startArgs, "--vm-driver=podman") || strings.Contains(*startArgs, "driver=podman")
 }
 
+// NeedsPortForward requires whether or not this host needs port forwarding
```

Comment:
coud maybe add more comments, for the contributors that dont know much about this.

docker on non-linux uses a bridge networking